src/message.ts

```
@@ -36,6 +43,46 @@ export class Message {
         }
     }
 
+    /**
+     * nack - not ack for a message, meaning that the message will be redelivered again to the same consumers group without waiting to its ack wait time.
+     */
+    nack() {
+        if (this.message.nak)
+            this.message.nak();
+        else
+            throw MemphisError(new Error('cannot nack DLS message'));
```

ignore the error in this case